radar/dev/vertx.yaml

```diff
@@ -1,27 +1,33 @@
 name: Vert.x
-logo: https://vertx.io/assets/logo-sm.png
+logo: https://upload.wikimedia.org/wikipedia/commons/3/3d/Vertx_logo.png
 blip:
   - date: 2018-03-20
     ring: TRIAL
+  - date: 2021-03-04
+    ring: ADOPT
 description: |
   Eclipse Vert.x is a polyglot event-driven application framework that runs on the
   Java Virtual Machine.
 
   Vert.x contains several different components designed to make it easier for
-  you to write compelling reactive applications in a range of different languages.
+  you to write reactive applications in a range of different languages.
   It is a compelling approach to writing microservices!
 
-  As of version 3.6.0, Vert.x exposes its API in Java, JavaScript, Groovy, Ruby,
-  Scala, Kotlin and Ceylon.
+  As of version 4.0.2, Vert.x exposes its API in Java, JavaScript, Groovy, Ruby,
+  Scala and Kotlin.
 rationale: |
   Vert.x is a very effective, highly performant framework where focus is on writing
   small isolated classes that communicates via a shared bus; EventBus.
```

Comment:
Should be changed to "classes that communicate"

Reply:
Fixed